qemu/tests/virtio_fs_share_data.py

```diff
@@ -541,7 +541,8 @@ def file_check(cmd):
                     error_context.context("Run fio on %s." % fs_dest, test.log.info)
                     fio = generate_instance(params, vm, 'fio')
                     try:
-                        fio.run(fio_options % guest_file, io_timeout)
+                        for bs in params.objects("stress_bs"):
```

Comment:
```suggestion
                        for bs in params.get_list("stress_bs"):
```
it's more fit to your requirements.

Reply:
Good suggestion, corrected